src/en/PinkMuffinTL.lua

```diff
@@ -1,4 +1,4 @@
--- {"id":4301,"ver":"1.0.9","libVer":"1.0.0","author":"MechTechnology"}
+-- {"id":4301,"ver":"1.1.0","libVer":"1.0.0","author":"MechTechnology"}

```

Comment:
The changes do not seem like a minor version bump to me.

Reply:
There is no standard for versioning, so i am not gonna mess with it since i already tested it as is. It does not really matter.